templates/Deepcell_Postprocessing.ipynb

```diff
@@ -131,10 +132,11 @@
     "# get input data for overlay\n",
     "input_data_xr = data_utils.load_imgs_from_multitiff(deepcell_input_dir, multitiff_files=points_input)\n",
     "\n",
-    "for fov in input_data_xr.fovs:\n",
-    "    plot_utils.plot_overlay(segmentation_labels.loc[fov, :, :, \"whole_cell\"].values,\n",
-    "                            input_data_xr.loc[fov, :, :, :].values,\n",
-    "                            path=os.path.join(label_dir, f'{fov.values}_overlay.tif'))"
+    "segmentation_utils.visualize_segmentation(\n",
+    "            segmentation_labels_xr=segmentation_labels,\n",
+    "            fovs=input_data_xr.fovs.values, channel_data_xr=input_data_xr,\n",
+    "            overlay_channels=[input_data_xr.channels.values[:2]],\n",
```

Comment:
You should provide overlay_channels as a list of strings with the actual channel names to make this easier to understand